### PR TITLE
feat: add text input component

### PR DESCRIPTION
**Pull request by corquaid**
Description
---
- Primitive `TextInput` component

Motivation and Context
---
issue #96 

How Has This Been Tested?
---
Unit tests

Screenshots:

https://user-images.githubusercontent.com/69508715/165788372-ed1cf8c5-e68c-4d8b-90fa-92a564843fe5.mov

Screenshot 2022-04-28 at 17 09 59
Screenshot 2022-04-28 at 17 09 51
Screenshot 2022-04-28 at 17 09 42
Screenshot 2022-04-28 at 17 09 34
Screenshot 2022-04-28 at 17 09 15
Screenshot 2022-04-28 at 17 08 31
Screenshot 2022-04-28 at 17 08 16
Screenshot 2022-04-28 at 17 10 38
Screenshot 2022-04-28 at 17 10 11

### CODE REVIEW

File: applications/launchpad_v2/src/styles/Icons/Password1.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    viewBox='0 0 22 22'
+    fill='none'
+    xmlns='http://www.w3.org/2000/svg'
+    {...props}
+  >
+    <path
+      d='M22 11A11 11 0 0 0 11 0v2.22A8.78 8.78 0 0 1 19.78 11H22Z'
+      fill='#A2281D'
+    />
+  </svg>
```
Comment (tomaszantas) [inline applications/launchpad_v2/src/styles/Icons/Password1.tsx:18]: You can add this icon to Icons tests.
You need to add here the `data-testid='svg-password1`.
And then, in `styles/Icons/Icons.test.tsx`, add `Password1` to:

```
const ICONS = [
  ...
  'Password1'
   ...
]
```

**Review by tomaszantas — CHANGES_REQUESTED** [review]
You can add tests for icons added in this PR (see the comment).

Also, not sure why we need to import the Smiley as `.svg` file to the `assets` folder, and then add also as the component in `styles/Icons`. (Just asking ;) )

File: applications/launchpad_v2/src/styles/Icons/Password1.tsx (hunk 1 above)
Comment (corquaid) [reply applications/launchpad_v2/src/styles/Icons/Password1.tsx:18]: unit tests added, images removed 👍

**Review by tomaszantas — APPROVED** [review]
LGTM!

**Pull request merged**